ansible_collections/arista/avd/roles/eos_cli_config_gen/schemas/schema_fragments/policy_maps.schema.yml

```diff
@@ -168,3 +168,33 @@ keys:
                           - "mbytes"
                           - "packets"
                         default: "bytes"
+      copp_system_policy:
+        type: dict
+        description: Control-plane policy configuration.
+        keys:
+          classes:
+            type: list
+            primary_key: name
+            items:
+              type: dict
+              keys:
+                name:
+                  type: str
+                shape:
+                  type: int
+                  description: Maximum rate limit.
+                  min: 0
+                  max: 10000000
+                  convert_types:
+                    - str
+                bandwidth:
+                  type: int
+                  description: Minimum bandwidth.
+                  min: 0
+                  max: 10000000
```

Comment:
I would expect this min and max to vary between pps and kbps. So we may have to move it to the description.

Reply:
hss044(config-s-claus-pmap-c-copp-copp-system-policy-copp-system-cvx)#shape kbps ?
  <0-10000000>  Shape value in the rate unit specified
  
  bko017(config-s-claus-pmap-c-copp-copp-system-policy-copp-system-cvx)#shape pps ?
  <0-10000000>  Shape value in the rate unit specified
  
  Here both units have same range.

Reply:
Does it vary with platform?

Reply:
hmm maybe not. Keep what you have. Thank you for checking.